File: RemoveElement/RemoveElement.java
package RemoveElement;

public class RemoveElement {
    public static int solution(int[] nums, int val) {
       int index = 0;
        for (int i = 0; i < nums.length; i++) {
            if (nums[i] != val) {
                nums[index] = nums[i];
                index++;
            }
        }
        return index;
    }

    public static void main(String[] args) {
        int[] nums = {3,2,2,3};
        System.out.println(solution(nums, 3));
    }
}

File: RemoveElement/RemoveElement.py
from typing import List


def removeElement(nums: List[int], val: int) -> int:
    index = 0
    for i in range(len(nums)):
        if (nums[i] != val):
            nums[index] = nums[i]
            index += 1
    return index


nums = [3,2,2,3]
val = 3

print(removeElement(nums,val))